ui/src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Redirect, Route } from 'react-router-dom';
```

Comment:
@laurafitzgerald what is the purpose of multiple aliases here?

Reply:
@davidffrench no purpose, will remove.

Reply:
I believe BrowserRouter is the import and `Router, Switch, Redirect, Route` are the aliases, this may not work any longer now.

Reply:
@davidffrench it actually turns out we need this alias here as we're using `react-router-dom` v4 See this [SO](https://stackoverflow.com/questions/43159975/cannot-read-property-location-of-undefined?rq=1) for details

Reply:
Thanks for the clarification @laurafitzgerald